src/__namespace.js

```diff
@@ -1,4 +1,5 @@
 /**
  * @ignore
  */
+window.wikibase = window.wikibase || {};
```

Comment:
Why does this use `window.`, but the line below does not? In my opinion this a) should be consistent, and b) avoid `window.` when not really necessary (e.g. certain headless JS engines might need it).

Reply:
Ping @manicki.

Reply:
It was needed in this line exactly to have phantom js understand where wikibase object is defined.
It was not apparently needed elsewhere. The whole weirdness is probably going to be irrelevant once this lib uses some proper way of registering its stuff (module.exports or whatever you do in JS this month).

Reply:
Ok, thanks!